packages/datatrak-web/src/views/LandingPage/RecentSurveysSection.tsx

```diff
@@ -44,25 +42,14 @@ const ScrollBody = styled.div`
 
 export const RecentSurveysSection = () => {
   const { data: recentSurveys = [], isSuccess, isLoading } = useCurrentUserRecentSurveys();
-  const navigate = useNavigate();
-  const { mutateAsync: editUser } = useEditUser();
-
-  const handleSelectSurvey = async (surveyCode: string, countryId: string) => {
```

Comment:
This is no longer needed as the countryId is checked and updated in the survey page now